internal/distro/rhel90/package_sets.go

```diff
@@ -113,7 +113,7 @@ func qcow2CommonPackageSet() rpmmd.PackageSet {
 			"@core", "authselect-compat", "chrony", "cloud-init",
 			"cloud-utils-growpart", "cockpit-system", "cockpit-ws",
 			"dhcp-client", "dnf", "dnf-utils", "dosfstools",
-			"insights-client", "NetworkManager", "net-tools", "nfs-utils",
```

Comment:
What previous definition do you mean?

Reply:
The ones made by Jozef that were generating v1 manifests.